data/brands/amenity/fast_food.json

```diff
@@ -2656,6 +2671,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Milio's",
```

Comment:
It seems like this should be the full name since they use Milio's Sandwiches on all the signage, social media and website.